src/screens/Dashboard/index.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import { useAppDispatch, useAppSelector } from 'hooks';
+import { SurveysState } from 'store/reducers/Surveys';
+import TestWrapper from 'tests/TestWrapper';
+
+import DashBoardScreen from '.';
+
+const mockUseNavigate = jest.fn();
+const mockDispatch = jest.fn();
+
+jest.mock('hooks');
+
+jest.mock('react-router-dom', () => ({
+  ...(jest.requireActual('react-router-dom') as jest.Mock),
```

Comment:
Do we need this 🤔

Reply:
@nvminhtue I think it's redundant. Let me check 🙆‍♂️

Reply:
@nvminhtue removed in [3aadb5c](https://github.com/manh-t/react-survey/pull/56/commits/3aadb5c4ed29ded049fb3ae88f102051848a7681)